books/free-programming-books-langs.md

```diff
@@ -1653,6 +1654,19 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [TrueOS® Users Handbook](https://www.trueos.org/handbook/trueos.html)
 
 
+### Pascal
+
+* [Free Pascal Reference guide](https://www.freepascal.org/docs-html/ref/ref.html)
+* [Modern Object Pascal Introduction for Programmers](https://castle-engine.io/modern_pascal_introduction.html) (HTML)
+* [Pascal Language Reference](https://docs.oracle.com/cd/E19957-01/802-5762/802-5762.pdf) (PDF)
+* [Pascal Programming Reference Manual](https://public.support.unisys.com/aseries/docs/clearpath-mcp-17.0/pdf/86000080-103.pdf) (PDF)
+* [Pascal Quick Reference](https://ksvi.mff.cuni.cz/~dingle/2017/pascal_reference.html)
+* [Turbo Pascal Reference Guide](http://bitsavers.org/pdf/borland/turbo_pascal/Turbo_Pascal_Version_5.0_Reference_Guide_1989.pdf) (PDF)
+* [Vector Pascal, an Array Language](http://www.dcs.gla.ac.uk/~wpc/reports/compilers/compilerindex/vp-ver2.html)
+* [Vector Pascal Reference Manual](https://www.researchgate.net/publication/220177664_Vector_Pascal_reference_manual) (PDF)
```

Comment:
In general, we do not link to file sharing sites, or file hosting sites such as google docs. or dropbox. (or researchgate)

Reply:
I can get rid of these sources - I thought they were good references, but I understand that file sharing/hosting is generally unreliable as long-term references.